tools/setup/emoji/emoji_names.py

```diff
@@ -1,1580 +1,2055 @@
+    "1f1f9-1f1e8": {"canonical_name": "flag: Turks & Caicos Islands", "aliases": []},
+    "1f1f9-1f1e9": {"canonical_name": "flag: Chad", "aliases": []},
+    "1f1f9-1f1eb": {"canonical_name": "flag: French Southern Territories", "aliases": []},
+    "1f1f9-1f1ec": {"canonical_name": "flag: Togo", "aliases": []},
+    "1f1f9-1f1ed": {"canonical_name": "flag: Thailand", "aliases": []},
+    "1f1f9-1f1ef": {"canonical_name": "flag: Tajikistan", "aliases": []},
+    "1f1f9-1f1f0": {"canonical_name": "flag: Tokelau", "aliases": []},
+    "1f1f9-1f1f1": {"canonical_name": "flag: Timor-Leste", "aliases": []},
+    "1f1f9-1f1f2": {"canonical_name": "flag: Turkmenistan", "aliases": []},
+    "1f1f9-1f1f3": {"canonical_name": "flag: Tunisia", "aliases": []},
+    "1f1f9-1f1f4": {"canonical_name": "flag: Tonga", "aliases": []},
+    "1f1f9-1f1f7": {"canonical_name": "flag: Turkey", "aliases": []},
+    "1f1f9-1f1f9": {"canonical_name": "flag: Trinidad & Tobago", "aliases": []},
+    "1f1f9-1f1fb": {"canonical_name": "flag: Tuvalu", "aliases": []},
+    "1f1f9-1f1fc": {"canonical_name": "flag: Taiwan", "aliases": []},
+    "1f1f9-1f1ff": {"canonical_name": "flag: Tanzania", "aliases": []},
+    "1f1fa-1f1e6": {"canonical_name": "flag: Ukraine", "aliases": []},
+    "1f1fa-1f1ec": {"canonical_name": "flag: Uganda", "aliases": []},
+    "1f1fa-1f1f2": {"canonical_name": "flag: U.S. Outlying Islands", "aliases": []},
+    "1f1fa-1f1f3": {"canonical_name": "flag: United Nations", "aliases": []},
+    "1f1fa-1f1f8": {"canonical_name": "flag: United States", "aliases": []},
+    "1f1fa-1f1fe": {"canonical_name": "flag: Uruguay", "aliases": []},
+    "1f1fa-1f1ff": {"canonical_name": "flag: Uzbekistan", "aliases": []},
+    "1f1fb-1f1e6": {"canonical_name": "flag: Vatican City", "aliases": []},
+    "1f1fb-1f1e8": {"canonical_name": "flag: St. Vincent & Grenadines", "aliases": []},
+    "1f1fb-1f1ea": {"canonical_name": "flag: Venezuela", "aliases": []},
+    "1f1fb-1f1ec": {"canonical_name": "flag: British Virgin Islands", "aliases": []},
+    "1f1fb-1f1ee": {"canonical_name": "flag: U.S. Virgin Islands", "aliases": []},
+    "1f1fb-1f1f3": {"canonical_name": "flag: Vietnam", "aliases": []},
+    "1f1fb-1f1fa": {"canonical_name": "flag: Vanuatu", "aliases": []},
+    "1f1fc-1f1eb": {"canonical_name": "flag: Wallis & Futuna", "aliases": []},
+    "1f1fc-1f1f8": {"canonical_name": "flag: Samoa", "aliases": []},
+    "1f1fd-1f1f0": {"canonical_name": "flag: Kosovo", "aliases": []},
+    "1f1fe-1f1ea": {"canonical_name": "flag: Yemen", "aliases": []},
+    "1f1fe-1f1f9": {"canonical_name": "flag: Mayotte", "aliases": []},
+    "1f1ff-1f1e6": {"canonical_name": "flag: South Africa", "aliases": []},
+    "1f1ff-1f1f2": {"canonical_name": "flag: Zambia", "aliases": []},
+    "1f1ff-1f1fc": {"canonical_name": "flag: Zimbabwe", "aliases": []},
+    "1f201": {"canonical_name": 'Japanese "here" button', "aliases": ['"here"\u30b3\u30b3']},
```

Comment:
I think the corresponding line for this in the exceptions file is missing a comma. An illustration of one reason why it's helpful to have the script take care of these fixes automatically where possible :slightly_smiling_face: